batch-setup/make_rawr_tiles.py

```diff
@@ -232,6 +258,26 @@ def make_rawr_tiles(rawr_config_file, missing_config_file, missing_bucket,
               "AWS_DEFAULT_REGION to be set.")
         sys.exit(1)
 
+    bbox_west = -181
```

Comment:
Please specify a valid bbox (within ±180, ±90), that is relatively small / not whole world. Like over Seattle / Western Washington state.

Reply:
> Please specify a valid bbox (within ±180, ±90), that is relatively small / not whole world. Like over Seattle / Western Washington state.

I will remove this section, I think people need to pass this information to trigger the bbox rebuild, rather than we rebuild a default bbox?